### PR TITLE
added comments to 

### PR DESCRIPTION
**Pull request by YashKumarVerma**
Fixes #

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:
- added comments

### CODE REVIEW

**Comment by raju249** [comment]
@YashKumarVerma Please add the issue number in the PR description which is solved by this PR.
Thanks

**Comment by harshit98** [comment]
Hi @YashKumarVerma. What's the status of this PR? Have you done the requested changes?

File: app/generate_csv_eventyay.py [hunk 1]
```diff
@@ -19,9 +21,12 @@ def tocsv(url_eventyay, filename):
     :param `filename` - Destination file name
     """
     with urlopen(url_eventyay) as url:
+        #load json, parse it using the utf charset
         data = json.loads(url.read().decode(url.info().get_param('charset') or 'utf8'))
+        # get list of speakers in separate vatiable
         speakers = data['speakers']
         with open(os.path.join(UPLOAD_FOLDER, filename), "w+") as f:
+        #process list one by one
```
Comment (yashLadha) [inline app/generate_csv_eventyay.py]: Try to rewrite it as `iterates the list of speakers`

**Comment by YashKumarVerma** [comment]
i'm on surface again. working on it 🔥

**Comment by YashKumarVerma** [comment]
closing to open again 😉

**Pull request closed without merging**